test/application_system_test_case.rb

```diff
@@ -63,4 +63,16 @@ def fill_autocomplete(name, options = {})
     end
     find("[name=\"#{name}\"]").native.send_keys :down, :enter
   end
+
+  def any_checked?(selector)
+    page.find_all(selector).reduce do |a, b|
+      a.checked? || b.checked?
+    end
+  end
+
+  def all_checked?(selector)
+    page.find_all(selector).reduce do |a, b|
+      a.checked? && b.checked?
+    end
```

Comment:
Not sure why you're using `reduce` here, you should be able to just do `.any?(&:checked)` and `.all?(&:checked)`

Reply:
As soon as you made me aware of this, I'm sure I want to use `.all?(&:checked?)` and `.any?(&:checked?)`! 🧀 🐭